libcnb-test/src/pack.rs

```diff
@@ -0,0 +1,92 @@
+use std::path::PathBuf;
+use std::process::Command;
+use tempfile::TempDir;
+
+/// Represents a `pack build` command.
+pub(crate) struct PackBuildCommand {
+    builder: String,
+    path: PathBuf,
+    image_name: String,
+    buildpacks: Vec<BuildpackReference>,
+    verbose: bool,
+}
+
+pub(crate) enum BuildpackReference {
+    Id(String),
+    Path(PathBuf),
+}
+
+impl From<PathBuf> for BuildpackReference {
+    fn from(path: PathBuf) -> Self {
+        BuildpackReference::Path(path)
+    }
+}
+
+impl From<&TempDir> for BuildpackReference {
+    fn from(path: &TempDir) -> Self {
+        BuildpackReference::Path(path.path().into())
+    }
+}
+
+impl From<String> for BuildpackReference {
+    fn from(id: String) -> Self {
+        BuildpackReference::Id(id)
+    }
+}
+
+impl PackBuildCommand {
+    pub fn new(
+        builder: impl Into<String>,
+        path: impl Into<PathBuf>,
+        image_name: impl Into<String>,
+    ) -> PackBuildCommand {
+        PackBuildCommand {
+            builder: builder.into(),
+            path: path.into(),
+            image_name: image_name.into(),
+            buildpacks: vec![],
+            verbose: false,
+        }
+    }
+
+    pub fn buildpack(&mut self, b: impl Into<BuildpackReference>) -> &mut Self {
+        self.buildpacks.push(b.into());
+        self
+    }
+}
+
+impl From<PackBuildCommand> for Command {
```

Comment:
It feels like some tests for this impl would be worthwhile?

Reply:
I added one in 0abc15e